modules/test/playwright/tests/frontend-data-set-web/customized.spec.ts

```diff
@@ -795,3 +795,162 @@ test(
 		});
 	}
 );
+
+test(
+	'Test quick action buttons function correctly',
+	{tag: '@LPS-153220'},
+	async ({fdsSamplePage, page}) => {
+		const firstItemActionsCell =
+			fdsSamplePage.table.itemActionsCells.first();
+
+		const thirdItemActionsCell =
+			fdsSamplePage.table.itemActionsCells.nth(2);
+
+		const firstItemActionButton = firstItemActionsCell.getByRole('button', {
+			exact: true,
+			name: 'Actions',
+		});
+
+		const thirdItemActionButton = thirdItemActionsCell.getByRole('button', {
+			exact: true,
+			name: 'Actions',
+		});
+
+		const quickActionPencilIcon = page
+			.getByRole('link', {name: 'Sample Edit'})
+			.nth(0);
+
+		const firstTableHeadCell = page.locator('.cell-select');
```

Comment:
As there is no `first()`, this is either incorrect, or misleadingly named. This should be `selectCells`, or `firstSelectCell` if you add in `first()`.